standalone/test/standaloneRoomViews_test.js

```diff
@@ -890,6 +890,20 @@ describe("loop.standaloneRoomViews", function() {
         });
       });
 
+      describe("Screen sharing paused", function() {
+        it("should display paused view if the screen share has been stopped", function() {
+          activeRoomStore.setStoreState({
+            roomState: ROOM_STATES.HAS_PARTICIPANTS,
+            streamPaused: true
+          });
+          clock.tick(loop.standaloneRoomViews.StandaloneRoomInfoArea.RENDER_WAITING_DELAY);
```

Comment:
You don't need the clock.tick here - the message is displayed straight away (its only certain cases we have a delay).